README.md

```diff
@@ -1,5 +1,127 @@
+    <img src="https://github.com/WoochanLee/BalanceTetris/blob/master/res/young.png?raw=true" height="300" />
+    <img src="https://github.com/WoochanLee/BalanceTetris/blob/master/res/chan.png?raw=true" height="300" />
+  </a>
+</p>
 
-Made by Woody
+<br>
 
-Play Tetris Link : [https://woochanlee.github.io/BalanceTetris/](https://woochanlee.github.io/BalanceTetris/)
+<p align="center">
+    <img src="https://github.com/WoochanLee/BalanceTetris/blob/master/res/logo.png?raw=true" width="600"/>
+</p>
+
+<p align="center" style="color: #000; font-size: 25px">🕹️You Can Single Play Online👇</p>
+<div align="center" style="margin-bottom: 40px">
+    <a href="https://github.com/WoochanLee/BalanceTetris" style="color: blue; font-size: 25px">
+        <u>https://github.com/WoochanLee/BalanceTetris</u>
+    </a>
+</div>
+
+<p align="center">
+    <img src="https://img.shields.io/badge/license-GPL%20v3-black.svg">
+    <img src="https://img.shields.io/badge/release-alpha-black.svg">
+<br />
+</div>
+
+<br>
+<br>
+
+# Features 🎮
+
+Easy to Play: You can play Tetris anywhere with a computer web browser.
+<br>
+Using T-spin: You can use T-spin technic.
+<br>
+Cats are cute: Cats are cute.
+<br>
+<br>
+<br>
+
+# How to Play 💁
+
+<b style="font-size: 15px">Q & E</b> or <b style="font-size: 15px">Z & C</b> --> Rotate the Control Block ( or you can use ↑ key )
+<br>
+<br>
+<b style="font-size: 20px">← ↓ →</b> --> Move the Control Block
+<br>
+<br>
+<b style="font-size: 15px">Space Bar</b> --> Lower the Control Block to the floor at once
+<br>
+<br>
+<b style="font-size: 15px">Left Shift</b> --> Save or eject blocks to temporary storage
+<br>
+<br>
+<br>
+
+# Contributing 🙋
+
+Do you want to contribute?
+<br>
+<b style="font-size: 15px">Great!</b> Just do Pull Request!
+<br>
+We will review your codes right away!
+<br>
+<br>
+Or if you have an opinion, please make an issue!
+<br>
+<br>
+<br>
+
+# Contributors 🧑‍💻
+
+Thanks to great developers 🎉
+
+<!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-disable -->
+<table>
+  <tr>
+    <td align="center">
+        <a href="https://github.com/DevSDK">
+            <img src="https://avatars.githubusercontent.com/u/18409763?s=460&u=575f88f1cdc9d6176aceb4b2a56217c4ff107fe1&v=4?s=128" style="border-radius: 64px" height="128px"/>
+            <br />
+            <sub style="font-size: 15px">
+                <b>Seokho Song</b>
+            </sub>
+        </a>
+    </td>
+    <br />
+    <td align="center">
+        <a href="https://github.com/oh3vci">
+            <img src="https://avatars.githubusercontent.com/u/20278777?s=460&u=1ba3da77f8f0d6bf12ba645a85805e4fe09dbeb2&v=4?s=64" style="border-radius: 64px" height="128px"/>
+            <br />
+            <sub style="font-size: 15px">
+                <b>Dae-ho Kim</b>
+            </sub>
+        </a>
+        <br />
+    </td>
+  </tr>
+</table>
+<br>
+<br>
+<br>
+
+# Author 🧑🏻‍💻
```

Comment:
I guess this is not required part because you are already noted as © Woochan Lee for copyright it indicates you are owner of this project.

I think we don't need to separate between author and contributor. How about place we as Authors?

Reply:
agree. PTAL!